Fwk/AppFwk/cafProjectDataModel/cafPdmUiCore/cafPdmUiOrdering.cpp

```diff
@@ -262,6 +262,14 @@ void PdmUiOrdering::add( const PdmObjectHandle* obj, LayoutOptions layout )
     m_ordering.push_back( std::make_pair( uiItem, layout ) );
 }
 
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+void PdmUiOrdering::addRowAppend( const PdmFieldHandle* field )
+{
```

Comment:
Not sure about the name here? Maybe `appendToRow` is better?

Reply:
Agree, changed name.